responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -1279,8 +1279,18 @@ def _get_feature_ranges(
                 res_object[_MIN_VALUE] = test[col].min()
                 res_object[_MAX_VALUE] = test[col].max()
             else:
-                min_value = float(test[col].min())
-                max_value = float(test[col].max())
+                col_min = test[col].min()
+                col_max = test[col].max()
+                try:
+                    min_value = float(col_min)
+                    max_value = float(col_max)
+                except Exception as e:
+                    raise ValueError(
```

Comment:
Could you change this to SystemErrorException?

Reply:
done.